uwblueprint/website-bp
# Implement SignOut with Google on Admin Portal

### Implementation description

- This PR contains the implementation of the sign out with google functionality to the admin portal. A function is created and hooked onto the logout button that runs the function when clicked.

![image](https://user-images.githubusercontent.com/63923219/201539958-bb76e16b-5ec3-4287-947b-cf0fca094c5b.png)
![image](https://user-images.githubusercontent.com/63923219/201539972-83b573ee-2a30-4042-ad29-b669de03aa11.png)

 Click on the logout button when signed into the admin portal. You should be taken to the login page as you have been signed out.

### Steps to test

1. Pull in this branch to your local.
2. Run on yoir localhost, and sign into the admin portal.
3. Click on the logout button, you should be taken to the login page, as sign-out is successful. 



### What should reviewers focus on?

- The logout button works as expected.

### Checklist
